pkg/service/verifycredential/revocation/revocation_service.go

```diff
@@ -114,5 +115,5 @@ func (s *Service) sendHTTPRequest(req *http.Request, status int, token string) (
 		return nil, fmt.Errorf("failed to read response body for status %d: %s", resp.StatusCode, string(body))
 	}
 
-	return body, nil
+	return bytes.Trim(body, "\n"), nil
```

Comment:
why we need trim here

Reply:
ctx.JSON(), ctx.String() and similar methods from echo adds `/n` to the end of payload body. 
For LDP VC it's ok, but for jwt it makes problems in decoding.